zerberus/zone.cpp

```diff
@@ -50,18 +50,19 @@ bool Zone::match(Channel* c, int k, int v, Trigger et)
          && (k <= keyHi)
          && (v >= veloLo)
          && (v <= veloHi)
-         && (seq == seqPos)
+         //&& (seq == seqPos)
```

Comment:
Oh and if you could remove this line, if it's not useful, better to not keep it commented.